CONTRIBUTING_ID.md

```diff
@@ -10,7 +10,6 @@
     - [Penugasan _issue_ & komunikasi](#penugasan-issue--komunikasi)
     - [Pembuatan _Draft Pull Request_](#pembuatan-draft-pull-request)
     - [Pemberian deskripsi _pull pequest_](#pemberian-deskripsi-pull-request)
-    - [Menonaktifkan GitHub Actions](#menonaktifkan-github-actions)
```

Comment:
Your PR keeps undoing the changes from the previous PRs. I'm wondering how you did it.

Please make sure that there's no changes other than your own changes reflected in this PR. Thanks!